python/lsst/pipe/tasks/computeExposureSummaryStats.py

```diff
@@ -206,6 +225,88 @@ def run(self, exposure, sources, background):
                                                 raCorners=raCorners,
                                                 decCorners=decCorners,
                                                 astromOffsetMean=astromOffsetMean,
-                                                astromOffsetStd=astromOffsetStd)
+                                                astromOffsetStd=astromOffsetStd,
+                                                nPsfStar=psfStats.nPsfStar,
+                                                psfStarDeltaE1Median=psfStats.psfStarDeltaE1Median,
+                                                psfStarDeltaE2Median=psfStats.psfStarDeltaE2Median,
+                                                psfStarDeltaE1Scatter=psfStats.psfStarDeltaE1Scatter,
+                                                psfStarDeltaE2Scatter=psfStats.psfStarDeltaE2Scatter,
+                                                psfStarDeltaSizeMedian=psfStats.psfStarDeltaSizeMedian,
+                                                psfStarDeltaSizeScatter=psfStats.psfStarDeltaSizeScatter,
+                                                psfStarScaledDeltaSizeScatter=scaledSize)
```

Comment:
Add the new entries to the class description?

Also, maybe that line should now read "...at the detector centers or averaged over a detector..."?